src/vaishnaviVJun20_arrayExamples/StringMethods.java

```diff
@@ -0,0 +1,36 @@
+package vaishnaviVJun20_arrayExamples;
+
+public class StringMethods {
+	
+	void findCharacterType(String s) {
+		int digitCount=0;
+		int lowerCaseCount=0;
+		int upperCaseCount=0;
+		int letterCount=0;
+		int specialCharCount=0;
+		for(int index=0;index<s.length();index++) {
+			if(Character.isDigit(s.charAt(index)))
+				digitCount++;
+			if(Character.isLetter(s.charAt(index)))
+				letterCount++;
+			if(Character.isUpperCase(s.charAt(index)))
+				upperCaseCount++;
+			if(Character.isLowerCase(s.charAt(index)))
```

Comment:
Here so many comparisons happening, which is not required. 
Like if char is digit so it will go in your all if block. which is wrong right !!
So here you need to write if, else if....else block. 
Second point letterCount without using this variable you can print the expected output. Think about it.

Reply:
Changes done, I wasn't thinking about that letter logic before that's why so many if's. Thanks :)